specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json

```diff
@@ -1035,6 +1264,14 @@
         "cors": {
           "$ref": "#/definitions/SignalRCorsSettings",
           "description": "Cross-Origin Resource Sharing (CORS) settings."
+        },
+        "upstream": {
+          "$ref": "#/definitions/ServerlessUpstreamSettings",
+          "description": "Upstream settings when the Azure SignalR is in server-less mode."
+        },
+        "networkACLs": {
+          "$ref": "#/definitions/SignalRNetworkACLs",
+          "description": "Network ACLs"
         }
```

Comment:
this needs to be a new api-version.  Handling it via "patch" semantics with nulls is not a good approach as that is not expected by consumers of your API (a PUT should replace all properties).  It makes what-if analysis on template deployments much more complex, will break users with custom Azure Policies in place that require certain properties, etc...